tests/testthat/test-unit-latent_individual.R

```diff
@@ -84,31 +109,36 @@ test_that("epidist_formula.epidist_latent_individual with custom formulas produc
 })
 
 test_that("epidist_formula.epidist_latent_individual with custom formulas errors for incorrect custom formulas", { # nolint: line_length_linter.
-  expect_error(epidist_formula(prep_obs, delay_central = ~ 1 + age))
-  expect_error(epidist_formula(prep_obs, sigma = ~ 1 + age))
-  expect_error(epidist_formula(prep_obs, delay_central = 1))
-  expect_error(epidist_formula(prep_obs, sigma = 1))
-})
-
-test_that("epidist_family.epidist_latent_individual with default settings produces an object of the right class", { # nolint: line_length_linter.
-  family <- epidist_family(prep_obs)
-  expect_s3_class(family, "customfamily")
-  expect_s3_class(family, "brmsfamily")
-  expect_s3_class(family, "family")
-})
-
-test_that("epidist_family.epidist_latent_individual warns users or gives an error when passed inappropriate family input", { # nolint: line_length_linter.
-  expect_error(epidist_family(prep_obs, family = 1))
-  expect_warning(epidist_family(prep_obs, family = "not_a_real_lpdf"))
-})
-
-test_that("the family argument in epidist_family.epidist_latent_individual passes as expected", { # nolint: line_length_linter.
-  family_gamma <- epidist_family(prep_obs, family = "gamma")
-  expect_equal(family_gamma$name, "latent_gamma")
+  expect_error(
```

Comment:
With these expect errors I don't like that they pass for any error. I know with the `regex` argument you can set the error you want. How to do this with `cli`?

Reply:
Could be a special `cli` `expect_error` wrapper